src/screens/SearchScreen.spec.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { __esModule } from "@testing-library/jest-dom/dist/matchers";
+import { render, waitFor, screen } from "@testing-library/react";
+import SearchScreen from "./SearchScreen";
+import "@testing-library/jest-dom";
+import { BrowserRouter } from "react-router-dom";
+import { LoadScript } from "@react-google-maps/api";
+
+describe("Basic functionality", () => {
+  __esModule: true,
+    it("should display header", async () => {
+      render(
+        <BrowserRouter>
```

Comment:
Why do we need browser router

Reply:
When we were using without BrowserRouter, then there was an error saying that "useNavigate cannot be used outside of Router". useNavigate is used while shifting from the main page to the search page. So to remove the error and for successful testing, we included BrowserRouter tag from react-router-dom.

Reply:
mock react-router-dom

Reply:
Updated